content/en/tracing/api_catalog/_index.md

```diff
@@ -1,18 +1,94 @@
+
+For example, click on the monitors section to open a detailed card with more information and a link to monitors for the endpoint.
+
+The performance graphs on the page are initially scoped to the same settings as on the API Catalog page, and you can change those settings in the details page  to suit your investigation by using the time frame selector and other scope drop-downs. 
+
+## Adding bulk tags
+
+In addition to tagging an endpoint in its details page, you can **add tags to multiple endpoints** at once in the API Catalog. Select multiple endpoint check boxes, and click **Edit tags** to provide ownership, importance, or other useful grouping information labels to the selected endpoints. This can help you quickly access groups of endpoints defined by your own criteria, and create assets such as monitors and dashboards from endpoints with specific tags.
+
+For example, if you want to create latency alerts for endpoints that are particularly sensitive to performance problems, tag those endpoints with a tag like `Latency sensitive`. Or tag endpoints that handle sensitive data with a tag like `PII` and alert when endpoints with that tag have a `401 Unauthorized` response. Other examples of useful tags:
+
+- Criticality level
+- `experimental`
+- `contains password`
+- `contains PII`
+- Business logic (for example: `payments` )
+- Specific customer group (`subscribers`)
+
+When you add a tag, it appears in the list of facets on the left of the catalog. Click a facet to filter the list and add the tag to the Search query field.
+
+## Exploring endpoints during incidents
+
+When investigating an incident, getting a higher resolution view through the API endpoints can lead to faster understanding of the root cause. Resolve incidents faster with **API ownership information** -- team, code repository, on-call details -- directly associated with each endpoint, to know who to reach when something goes wrong. 
+
+The ownership information displayed on API Catalog pages is derived from the underlying Service definition supplied to the Service Catalog.
+
+
+## Creating monitors
+
+I want to get alerted each time an endpoint (or a group of endpoints) with high throughput gets a high error rate, so that I will not get false positive alerts on high error rate with low throughput.
+As a team leader, I want to monitor all the endpoints my team owns in a specific way that fits my team.
+As an SRE, I would like to list the defined Monitors of endpoints I care about, to make sure they are less of a risk in case of an incident.
+As an SRE, I would like to view all endpoints with defined Monitors (alerted), so I can address their endpoint owner so they can fix the alert.
+
+
+choose the type of metric in advance from the catalog (since there would be a lot of not-API-related queries inside the monitors page) and it would be exported together with the defined queries
+
+## Setting up
+
```

Comment:
Automatically built on top of APM data , API Catalog can be turned on with no additional setup required.

Only APM instrumented services and supported APM integrations are discoverable (link to FAQs or have a section about the supported integrations)

FAQs in Conf - https://datadoghq.atlassian.net/wiki/spaces/APIM/pages/2721185795/FAQ